src/Utils.ts

```diff
@@ -957,6 +973,33 @@ export class DummyInputManager {
             }
         }
     }
+
+    static addPhantomDummyWithTarget(
+        tabster: Types.TabsterCore,
+        anchor: HTMLElement,
+        isBackward: boolean,
+        element: HTMLElement
```

Comment:
what is `element` in this context?

Reply:
I can rename it to targetElement.

Reply:
can you describe the relationship between `anchor` and `targetElement`? I still can't quite understand it

Reply:
Anchor is the element we're moving from. Practically, the currently focused element. The phantom dummy will be inserted before or after this element (depending on the direction of travel and I am fixing a bug that for the cases with nested focusables it sometimes should be inserted inside the anchor element. Basically, it should be physically the element the browser will pick to focus if default action for Tab press is allowed. If something that controls the uncontrolled area picks an element to focus, so be it, the phantom dummy will just disappear with the next tick. But if it allows the default action, our dummy will receive focus and transfer it to the target element (and of course will also disappear with the next tick).

Reply:
Renamed it to sourceElement and targetElement.